orchestrator/orchestratorAutomation.ps1

```diff
@@ -30,36 +34,66 @@ Param (
    [Parameter(Mandatory = $false, ValueFromPipelineByPropertyName)]
    [string] $aifport,
    [Parameter(Mandatory = $false, ValueFromPipelineByPropertyName)]
-   [string] $domainBasedAccess,
+   [string] $portlessAccess,
    [Parameter(Mandatory = $false, ValueFromPipelineByPropertyName)]
    [string] $storageport
 )
 
-Import-Module 'WebAdministration'
-    if(!$config){   
-        $config = "C:\Program Files (x86)\UiPath\Orchestrator\web.config"
-    }
+Import-Module 'WebAdministration' 
+    
 
-    if(!$aifport){   
-        $aifport = "31390"
-    }
+if(!$config){   
+    $config = "C:\Program Files (x86)\UiPath\Orchestrator"

```

Comment:
why is config folder location?

Reply:
let's call it baseFolder

Reply:
it's just a cosmetic change vivek , if we modify it, we have to change the documentation as well...